src/widgets/helper/ChannelView.cpp

```diff
@@ -1316,13 +1316,24 @@ void ChannelView::mouseMoveEvent(QMouseEvent *event)
         {
             auto element = &hoverLayoutElement->getCreator();
             auto thumbnailSize = getSettings()->thumbnailSize;
-            if (thumbnailSize == 0 || isInStreamerMode())
+            if (thumbnailSize == 0)
```

Comment:
Can be simplified.
```suggestion
            if (!thumbnailSize)
```

Reply:
I don't really mind this being more explicit